test/e2e/persistent_volumes-vsphere.go

```diff
@@ -137,7 +141,6 @@ var _ = framework.KubeDescribe("PersistentVolumes:vsphere", func() {
 	It("should test that deleting a PVC before the pod does not cause pod deletion to fail on PD detach", func() {
 		By("Deleting the Claim")
 		framework.DeletePersistentVolumeClaim(c, pvc.Name, ns)
-
 		pvc, err = c.CoreV1().PersistentVolumeClaims(ns).Get(pvc.Name, metav1.GetOptions{})
```

Comment:
it's unnecessary to check that the pvc is deleted.  `DeletePersistentVolumeClaim` will end the test if there was an error during the deletion other than `isNotFound`.